pkg/operator/k8sutil/node_test.go

```diff
@@ -67,3 +67,5 @@ func TestValidNode(t *testing.T) {
 	validNodes := GetValidNodes(nodes, clientset, placement)
 	assert.Equal(t, len(validNodes), 1)
 }
+
+/* TODO: more tests */
```

Comment:
more tests will be included in this PR?

Reply:
I still have more tests to add, but I did get a couple more in.